.github/scripts/tkn_check_parse.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# removes `actions's` default -e flag as it breaks the logic
+set +e
+
+setup() {
+    curl -LO "https://github.com/tektoncd/cli/releases/download/v${TKN_CLI_VERSION}/tektoncd-cli-${TKN_CLI_VERSION}_Linux-64bit.deb" &>/dev/null
+    sudo dpkg -i ./tektoncd-cli-${TKN_CLI_VERSION}_Linux-64bit.deb >/dev/null
+}
+
+is_yaml() {
+    FILE=`basename $1`
+    DIRNAME=`dirname $1`
+    [ ${DIRNAME##*/} == "workflows" ] && return 1
+    [ "${FILE#*.}" == "yaml" ] && return 0 || return 1
+}
+
+main() { 
+  TKN="tkn"
+  VER=`${TKN} version |head -1`
+  REGISTRY="foo.bar/foo:bar"
+  TMPFILE=$(mktemp)
+
+  echo -e "${VER}\n"
```

Comment:
We save the version only to print it later? Why not just do `tkn version |head -1` and be done with it? But also, we specifically install a given version, so do we really need to check if it's really installed?